docs/samples/client/oauth/csharp/UnbrandedCredential.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Also, I discussed with Laurent and we agreed that it would be better to have the language as the root directory under client - samples/client/csharp/oauth